commands/audit/audit.go

```diff
@@ -171,10 +179,7 @@ func RunAudit(auditParams *AuditParams) (results *xrayutils.Results, err error)
 		errGroup.Go(utils.DownloadAnalyzerManagerIfNeeded)
 	}
 
-	if auditParams.xrayGraphScanParams.XscGitInfoContext != nil {
-		if err = xrayutils.SendXscGitInfoRequestIfEnabled(auditParams.xrayGraphScanParams, xrayManager); err != nil {
-			return nil, err
-		}
+	if auditParams.xrayGraphScanParams.MultiScanId != "" {
 		results.MultiScanId = auditParams.xrayGraphScanParams.MultiScanId
 	}
```

Comment:
```suggestion
	results.MultiScanId = auditParams.MultiScanId
```
MultiScanId should be stored at auditParams, at this point of adding MSI to every audit scan we can just set the value from the params to the one at the result